.gitmodules

```diff
@@ -944,3 +944,6 @@
 [submodule "libraries/drivers/acep7in"]
 	path = libraries/drivers/acep7in
 	url = https://github.com/adafruit/Adafruit_CircuitPython_ACeP7In.git
+[submodule "libraries/drivers/wii_classic"]
+	path = libraries/drivers/wii_classic
+	url = git@github.com:adafruit/Adafruit_CircuitPython_Wii_Classic.git
```

Comment:
The other repos use the HTTPS version of the link, I think it might be good to remain consistent with that

Reply:
ah okay! will change 🙂